server/channels/app/post_persistent_notification_test.go

```diff
@@ -216,6 +217,6 @@ func TestSendPersistentNotifications(t *testing.T) {
 	_, appErr := th.App.CreatePost(th.Context, p1, th.BasicChannel, model.CreatePostFlags{})
 	require.Nil(t, appErr)
 
-	err := th.App.SendPersistentNotifications()
-	require.NoError(t, err)
+	err = th.App.SendPersistentNotifications().(*model.AppError)
+	require.Nil(t, err)
```

Comment:
That seems wrong. `th.App.SendPersistentNotifications` returns an `error`. Why can't you check that?

Reply:
Getting no error from my side:
![image](https://github.com/user-attachments/assets/e6897d65-76b2-4577-846e-211c88e09d2c)

Reply:
This was a lil bit fishy. It was throwing error before, so I changed it to `appErr`. But now, I reverted it back, and it's not panicking.